### DISCUSSION TITLE
WMO Alerts Provider

### DISCUSSION THREAD

**Discussion by Joe-D01** [breezy-weather/breezy-weather#1557] [Q&A] [answered]
I wanted to ask how you are utilizing the WMO to fetch alerts. I couldn’t find any API endpoints related to this.

**Comment by papjul** [comment]
1. Tap the Pencil icon on top right of your location
2. Tap "Weather sources"
3. Tap "Alerts" (scroll a bit if you can't see it)
4. Choose "WMO Severe Weather Information Centre"
5. Save

FYI, WMO doesn't emit alerts on its own, they are just an intermediate to national sources.

**Reply by Joe-D01** [reply]
No, I mean, how does the app do it? Is it web scraping or something?

**Reply by papjul** [reply, marked as answer]
If you are looking for the source code, it's here: https://github.com/breezy-weather/breezy-weather/tree/main/app/src/src_nonfreenet/org/breezyweather/sources/wmosevereweather

Remember to respect the LGPL-v3 license and credit Breezy Weather if you re-use parts of the logic in a project ;)
